client/src/components/History/CurrentHistory/HistoryOperations/SelectionOperations.vue

```diff
@@ -119,28 +119,28 @@ export default {
     },
     methods: {
         // Selected content manipulation, hide/show/delete/purge
-        async hideSelected() {
-            await this.runOnSelection(hideSelectedContent);
+        hideSelected() {
+            this.runOnSelection(hideSelectedContent);
         },
-        async unhideSelected() {
-            await this.runOnSelection(unhideSelectedContent);
+        unhideSelected() {
+            this.runOnSelection(unhideSelectedContent);
         },
-        async deleteSelected() {
-            await this.runOnSelection(deleteSelectedContent);
+        deleteSelected() {
+            this.runOnSelection(deleteSelectedContent);
         },
-        async undeleteSelected() {
-            await this.runOnSelection(undeleteSelectedContent);
+        undeleteSelected() {
+            this.runOnSelection(undeleteSelectedContent);
         },
-        async purgeSelected() {
-            await this.runOnSelection(purgeSelectedContent);
+        purgeSelected() {
+            this.runOnSelection(purgeSelectedContent);
         },
         async runOnSelection(operation) {
-            this.$emit("update:operation-running", true);
+            this.$emit("update:operation-running", this.history.update_time);
             const items = this.getExplicitlySelectedItems();
             const filters = getQueryDict(this.filterText);
             this.$emit("update:show-selection", false);
             await operation(this.history, filters, items);
```

Comment:
By the way... this is totally unrelated to this PR but I might need these changes merged to avoid potential conflicts.

I figured out that now we need to do a bit of error handling after the bulk operation. If for some reason, every item in the request fails (`success_count: 0`) there will be no history update and we don't want to wait for that otherwise we will get the *processing...* message until the next history update which may or may not happen any time soon.

So we will have to avoid waiting for the History changes in that situation and possibly also display an error dialog with the details. There is also the possibility of a *partial* success in which some items are processed ok, but others fail for any reason... in this situation, we still want to wait for the History changes, but we also may want to display the error dialog with the ones that failed.

Any recommendation for handling that?